paas_app_charmer/charm.py

```diff
@@ -194,6 +196,15 @@ def _on_config_changed(self, _event: ops.EventBase) -> None:
         """
         self.restart()
 
+    @block_if_invalid_config
+    def _on_secret_changed(self, _event: ops.EventBase) -> None:
```

Comment:
Is `_event` deliberately named to be private? Is that to indicate that it's unused? If so, the convention (as far as I have ever seen) is to use just `_` by itself. `_event` seems odd.

Reply:
Yes, you are right, they are not used in the function, I have updated all of them to `_`, thanks!